pkg/cmd/token_validator.go

```diff
@@ -48,7 +48,8 @@ func (v *JWTValidatorImpl) Validate(context *config.Context) error {
 		return errors.New("malformed token: no expiration")
 	}
 
-	if float64(v.Clock.Now().Unix()) > exp {
+	// Add a time buffer of 1 minute to the token validator
+	if float64(v.Clock.Now().Unix()+60) > exp {
```

Comment:
Add `time.Minute` to the date instead of 60? (and then call `.Unix()`)

Reply:
```suggestion
	if float64(v.Clock.Now().Add(time.Minute).Unix()+60) > exp {
```
Based on Brian's suggestion. You'll also need to import `"time"` at the top.